music_assistant/controllers/media/playlists.py

```diff
@@ -30,13 +33,14 @@ class PlaylistController(MediaControllerBase[Playlist]):
     media_type = MediaType.PLAYLIST
     item_cls = Playlist
 
-    def __init__(self, *args, **kwargs) -> None:
+    def __init__(self, mass: MusicAssistant) -> None:
         """Initialize class."""
-        super().__init__(*args, **kwargs)
+        super().__init__(mass)
         # register (extra) api handlers
         api_base = self.api_base
         self.mass.register_api_command(f"music/{api_base}/create_playlist", self.create_playlist)
-        self.mass.register_api_command("music/playlists/playlist_tracks", self.tracks)
+        # TODO: This is a type mismatch that requires a logic change to fix properly
```

Comment:
didnt you fix this already in the api helper ?

Reply:
Were you thinking of this similar question? https://github.com/music-assistant/server/pull/2627#discussion_r2550500945